### PR TITLE
testing changes to secrets

### PR DESCRIPTION
**Pull request by rawanbadawi**

### CODE REVIEW

File: terraform/modules/secrets/README.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+- [HCA Cloud Module](#hca-cloud-module)
```
Comment (chasinandrew) [inline terraform/modules/secrets/README.md:1]: Can we remove all references to HCA?

File: terraform/modules/secrets/LICENSE
Comment (chasinandrew) [inline terraform/modules/secrets/LICENSE:1]: Please delete this file

File: terraform/modules/secrets/tests/defaults/main.tf [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  }
+}
+
+#{ module_testing_block }#
+
+module "secret_manager" {
+  for_each = local.secret_instances
+  source   = "../.."
+
+  # source  = "app.terraform.io/hca-healthcare/secret-manager/gcp"
```
Comment (chasinandrew) [inline terraform/modules/secrets/tests/defaults/main.tf]: Please remove this line referencing hca

Comment (rawanbadawi) [reply terraform/modules/secrets/tests/defaults/main.tf]: i removed all tests

**Review by chasinandrew — APPROVED** [review]

**Pull request merged**